README.md

```diff
@@ -19,8 +19,8 @@ This bundle allows to submit PubSub events via HTTP/POST requests to a [Crossbar
 
 ## Requirements
 * PHP >= 7
-* Guzzle 5 or 6
-* The following Symfony components (or the full-stack framework), version 2.7, 2.8, 3.x or 4.x:
+* Guzzle 5, 6 or 7 (7 is recommended)
+* The following Symfony components (or the full-stack framework), version 4.x, 5.x, 6.x, or 7.x:
```

Comment:
Do we support 7.x? It's not tested in the CI then.

Reply:
The tests with the highest (I'm going to fix the wording) use Guzzle 7 indeed.